x-pack/libbeat/autodiscover/providers/aws/elb/fetch.go

```diff
@@ -0,0 +1,220 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package elb
+
+import (
+	"context"
+	"sync"
+
+	"github.com/aws/aws-sdk-go-v2/service/elasticloadbalancingv2/elasticloadbalancingv2iface"
+
+	"github.com/elastic/beats/libbeat/logp"
```

Comment:
Nit. Our own packages should be at the end on imports, this import block should be something like:
```
import (
	"context"
	"sync"

	"github.com/aws/aws-sdk-go-v2/service/elasticloadbalancingv2"
	"github.com/aws/aws-sdk-go-v2/service/elasticloadbalancingv2/elasticloadbalancingv2iface"
	"go.uber.org/multierr"

	"github.com/elastic/beats/libbeat/logp"
)